docs/recipes/Code-Splitting-Using-Holocron.md

```diff
@@ -1,7 +1,240 @@
+[holocron-readme]: https://github.com/americanexpress/holocron/tree/master/packages/holocron/README.md
+[holocron-module-route-readme]: https://github.com/americanexpress/holocron/tree/master/packages/holocron-module-route/README.md
+[one-app-bundler-readme]: https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler/README.md
+[one-app-router-readme]: https://github.com/americanexpress/one-app-router/README.md
+[frank-lloyd-root]: ../../prod-sample/sample-modules/frank-lloyd-root/0.0.0/README.md
+[franks-burgers]: ../../prod-sample/sample-modules/franks-burgers/0.0.0/README.md
+
 [👈 Return to Overview](./Recipes.md)
 
 # Code Splitting Using Holocron
 
-Recipe is forthcoming.
+For One App, Holocron modules are a formal way of code-splitting for a modern,
+micro front-end in the One App ecosystem. Utilizing the module map as a system
+which promotes synergy and asynchrony across teams and collaborators on
+a single application.
+
+With Holocron each individual module is by definition, a code split bundled
+component with language packs that can be combined and composed based on our
+needs. Each module is versioned, configurable, routable and can be further
+split into smaller chunks using dynamic importing.
+
+Like `@loadable/component` and `react-loadable`, `holocron` is a complete code-splitting
+solution that works in both client and server side, however there is no need to scaffold
+a build infrastructure to support code-splitting when we use `one-app-bundler`.
```

Comment:
What about explaining the performance benefits? Smaller bundle size, faster time to first paint potentially? Maybe referencing React Docs on their views on performance gains for Code Splitting?

Also, maybe explain why you would Code Split on a Web Experience level (Holocron Module) versus Chunking a Module (Lazy Import).

Reply:
great points, I'll create a performance section to talk about these concepts

Reply:
updated in cb5e958, I've added both advantages and costs/considerations